### PR TITLE
epic squad hospital corpsman gameplay - adds pocket-fitting first-aid pouches to cargo

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
Adds `CREW_PAYCHECK * 6` (300 cr at time of writing) first-aid pouches to Cargo's goodies tab. These fit in pockets and come in two varieties; wound stabilizing and mini-medkit. however, they can be repacked at the end-user's leisure (still only fitting medical-related objects).

Sprites taken from TGMC's first-aid pouches.

Both come with:
- an emergency first-aid pen
- gauze (for bleeds/burns)

The wound stabilizing setup comes with:
- cautery
- bonesetter

while the mini-medkit comes with:
- sutures
- regenerative mesh

Also adds an adminspawn-only medkit pouch and generic storage pouch.
Also repaths the medpen pouch, materials pouch, and ammo pouch to the new pouch subtype to make them not have action buttons.
## How This Contributes To The Skyrat Roleplay Experience
bootleg IFAK jumpscare

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>
  
image
image
image

</details>

## Changelog

:cl:
add: Cargo now stocks pocket-clip-equipped personal first-aid pouches in their goodies tab for the low, low price of 6 crew paychecks (300 cr at the time of writing).
code: The pouch path now exists in the code, for pocket-fitting storage items that don't require the item-scooping that proper bags have.
/:cl:

### CODE REVIEW

**Comment by burgerenergy** [comment]
imo; too many storage slots. it turns your pocket into slightly worse medical belt and you can have two besides. Medical already has a huge variety of storage methods, this would be a first aid pouch for non-medical people. It being a little baggie with gauze/pen/maaaaybe-sutures would be better, as other things feel like they should be in medical's purview.

**Comment by RatFromTheJungle** [comment]
paramedics should start with one of these

**Comment by YakumoChen** [comment]
Seems like just a straight up normal medkit but better?

**Comment by Hatterhat** [comment]
> imo; too many storage slots. it turns your pocket into slightly worse medical belt and you can have two besides. Medical already has a huge variety of storage methods, this would be a first aid pouch for non-medical people. It being a little baggie with gauze/pen/maaaaybe-sutures would be better, as other things feel like they should be in medical's purview. 

i bumped it down to 4 slots and i could probably make it only fit in one pocket if its that egregious tbh

File: modular_skyrat/modules/modular_items/code/bags.dm [hunk 1]
```diff
@@ -1,40 +1,131 @@
...
+	. = ..()
+	var/static/items_inside = list(
+		/obj/item/stack/medical/gauze = 1,
+		/obj/item/stack/medical/splint = 1,
+		/obj/item/stack/medical/suture = 2,
+		/obj/item/stack/medical/mesh = 2,
+		/obj/item/reagent_containers/hypospray/medipen = 1,
+		/obj/item/healthanalyzer = 1,
+	)
+	generate_items_inside(items_inside,src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/bags.dm]:
```suggestion
	generate_items_inside(items_inside, src)
```

File: modular_skyrat/modules/modular_items/code/bags.dm [hunk 2]
```diff
@@ -1,40 +1,131 @@
...
+/obj/item/storage/pouch/medical/firstaid/loaded/Initialize(mapload)
+	. = ..()
+	desc += " Repackaged with station-standard medical supplies."
+	var/static/items_inside = list(
+		/obj/item/stack/medical/suture = 1,
+		/obj/item/stack/medical/mesh = 1,
+		/obj/item/stack/medical/gauze/twelve = 1,
+		/obj/item/reagent_containers/hypospray/medipen/ekit = 1,
+	)
+	generate_items_inside(items_inside,src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/bags.dm]:
```suggestion
	generate_items_inside(items_inside, src)
```

File: modular_skyrat/modules/modular_items/code/bags.dm [hunk 3]
```diff
@@ -1,40 +1,131 @@
...
+/obj/item/storage/pouch/medical/firstaid/stabilizer/Initialize(mapload)
+	. = ..()
+	desc += " Repackaged with a wound stabilization-focused loadout."
+	var/static/items_inside = list(
+		/obj/item/cautery = 1,
+		/obj/item/bonesetter = 1,
+		/obj/item/stack/medical/gauze/twelve = 1,
+		/obj/item/reagent_containers/hypospray/medipen/ekit = 1,
+	)
+	generate_items_inside(items_inside,src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/bags.dm]:
```suggestion
	generate_items_inside(items_inside, src)
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/modular_items/code/bags.dm [hunk 4]
```diff
@@ -1,40 +1,131 @@
...
+/obj/item/storage/pouch/medical/loaded/Initialize(mapload)
+	. = ..()
+	var/static/items_inside = list(
+		/obj/item/stack/medical/gauze = 1,
+		/obj/item/stack/medical/splint = 1,
+		/obj/item/stack/medical/suture = 2,
+		/obj/item/stack/medical/mesh = 2,
+		/obj/item/reagent_containers/hypospray/medipen = 1,
+		/obj/item/healthanalyzer = 1,
+	)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/bags.dm]: Isn't this 8 items, when you made the max amount of slots be 7?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/modular_items/code/bags.dm [hunk 5]
```diff
@@ -1,40 +1,130 @@
...
+	atom_storage.max_specific_storage = WEIGHT_CLASS_NORMAL
+	atom_storage.max_slots = 7
+	atom_storage.max_total_storage = 14
+	atom_storage.set_holdable(med_pouch_holdables)
+
+/obj/item/storage/pouch/medical/loaded/Initialize(mapload)
+	. = ..()
+	var/static/items_inside = list(
+		/obj/item/stack/medical/gauze/twelve = 1,
+		/obj/item/stack/medical/splint = 1,
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/bags.dm]: Splints aren't a thing anymore (even if I don't think that was necessarily the good thing to do), so you'll have to remove them from here.

**Review by GoldenAlpharex — DISMISSED** [review]

File: modular_skyrat/modules/modular_items/code/bags.dm (hunk 5 above)
Comment (Hatterhat) [reply modular_skyrat/modules/modular_items/code/bags.dm]: oh yeah i forgot that pr got merged

**Review by lessthnthree — APPROVED** [review]

**Pull request merged**